WcaOnRails/app/views/mail_form/dob_contact.erb

```diff
@@ -17,6 +17,13 @@
       <td>Correct birthdate</td>
       <td><strong><%= @resource.dob %></strong></td>
     </tr>
+    <tr style="height: 1em">
+      <td colspan="2"></td>
+    </tr>
+    <tr>
+      <td>Incorrect WCA ID claims</td>
+      <td><strong><%= @resource.incorrect_wca_id_claim_count %></strong></td>
```

Comment:
Maybe put a `/ #{User::MAX_INCORRECT_WCA_ID_CLAIM_COUNT}` in here?

Reply:
Great idea, done!